ADR/0011-managed-workspace-tier.md

```diff
@@ -0,0 +1,54 @@
+# 11. Managed Workspace Tier
+
+Date: 2023-01-09
+
+## Status
+
+New
+
+## Context
+
+Stonesoup provides the ability to release content to privileged destinations.
+For instance, to a production environment managed by a team that is not the same as the service team (as in the case of a centralized operations or centralized SRE team), or, for another example, to a production registry managed by a release engineering team (as in releasing to registry.redhat.io)
+
+This is enabled via flexible release pipelines that are curated by the Stonesoup release team.
+These pipelines include the ability to run tasks using secrets for an organization that are needed to access privileged destinations.
+
+These pipelines are triggered by the Release Service based on instances of Release custom resources. They are configured to execute in managed workspaces. This workspace type is one that is managed in the sense that a dedicated team or organization (SRE or Software production) controls access to the workspace.
+This restricted access is needed since this workspace type houses Production environments/services.
```

Comment:
Are you talking about setting roles (with specific privileges) for users that have access to the workspace? 
https://docs.google.com/document/d/1KMwHbvTAJ-AVKdHo-gVq_LkgpmwKKdCtCiI04X69kII/edit

Reply:
or in https://github.com/redhat-appstudio/book/pull/42/files instead of that document